### PR TITLE
Add documentation on enabling the Java Heap Profiler

### PR DESCRIPTION
**Pull request by luhenry**
### What does this PR do?

Add instructions on enabling the Java Heap Profiler.

### Motivation

We want to release the Java Heap Profiler in Beta. We need to guide customers to instructions on how to do so.

### Preview

https://docs-staging.datadoghq.com/luhenry/heap-profiler/tracing/profiler/profiler_troubleshooting

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/profiler/profiler_troubleshooting.md [hunk 1]
```diff
@@ -61,6 +61,20 @@ jdk.ObjectAllocationOutsideTLAB#enabled=true
...
+
+To enable the heap profiler, start your application with the `-Ddd.profiling.heap.enabled=true` JVM setting or the `DD_PROFILING_HEAP_ENABLED=true` environment variable.
+
+Alternatively, you can enable the following events in your `jfp` [override template file](#creating-and-using-a-jfr-template-override-file):
+
+```
+jdk.OldObjectSample#enabled=true
+```
+
+There is no support for reference chains at this time. Please [open a support ticket][2] if it's something you are interested in.
```
Comment (jbachorik) [inline content/en/tracing/profiler/profiler_troubleshooting.md:74]: I am not sure we should be mentioning 'reference chains' as it is a rather implementation specific term and most of the users would not have any idea what it is.

Comment (luhenry) [reply content/en/tracing/profiler/profiler_troubleshooting.md:74]: I would still want to gather a pulse from customers whether they are interested in the feature. @GaurabAryal where do you think is the best place to gather feedback and interest on it?

File: content/en/tracing/profiler/profiler_troubleshooting.md [hunk 2]
```diff
@@ -61,6 +61,20 @@ jdk.ObjectAllocationOutsideTLAB#enabled=true
 
 [Learn how to use override templates.](#creating-and-using-a-jfr-template-override-file)
 
+## Enabling the heap profiler
+
```
Comment (kayayarai) [inline content/en/tracing/profiler/profiler_troubleshooting.md:65]: If this is beta, we should add a statement about that here. Confirm and I will provide some wording.

File: content/en/tracing/profiler/profiler_troubleshooting.md (hunk 1 above)
Comment (kayayarai) [inline content/en/tracing/profiler/profiler_troubleshooting.md:74]:
```suggestion
[Contact Support][2] if you have feedback or feature requests for the Java heap profiler.
```

Comment (kayayarai) [reply content/en/tracing/profiler/profiler_troubleshooting.md:74]: I added a suggested rewording. Let me know what you think.

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/tracing/profiler/profiler_troubleshooting.md (hunk 2 above)
Comment (luhenry) [reply content/en/tracing/profiler/profiler_troubleshooting.md:65]: Yes, it's beta. Let's add a disclaimer.

Comment (kayayarai) [reply content/en/tracing/profiler/profiler_troubleshooting.md:65]:
```suggestion
<div class="alert alert-info">The Java heap profiler feature is in beta.</div>
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**